### ISSUE TITLE
Community involvement

### ISSUE THREAD

**Issue by Jogai** [almende/vis#4015]
Its good that the project asks for help from the community, but I think the project would benefit if the maintainers looked at pr's more often. There are some issues labeled "Fixed, waiting for release". It might be better to do more small releases to keep the community interested in contributing.

There are more pr opened than closed, the same with issues. The maintainers are barely commenting on anything. Personally that doesn't inspire me to make more contributions...

**Comment by dev-warner** [comment]
👍 we've got a fork for our project at work looking to contribute our changes in the near future, it would be great if there more active releases but understand it must be hard to juggle time.

**Comment by avrahamcool** [comment]
any response from the maintainers?
there are a lot of fixes already committed (and verified) that are crucial for vis.
when do you plan on releasing a new version?

**Comment by Jogai** [comment]
@yotamberk created his own fork, but it seems he's mostly focusing on timeline. I like forks better if they get an 'official endorsement' from the main project. Now its not clear what the other maintainers ( @ludost @mojoaxel @Tooa @eymiha @bradh @wimrijnders @macleodbroad-wf ) think about the direction of the project.

**Comment by mojoaxel** [comment]
It looks like @yotamberk is the only one still involved in the project.
I would consider this project **unmaintaind** for the moment.

The only way to get this project back on his feed would be to find a sponsor ([opencollective](https://opencollective.com/vis)). The first thing a maintanier should do (in my opinion) would be to the library in multible modules (network, timeline..),  and to ensure a fast release-process.

**Comment by bradh** [comment]
I previously retired from maintainership - I couldn't handle the "entitled" approach I saw from a small number of users.

**Comment by katjasmin** [comment]
Well it would be helpful if the opencollective link would be more prominent. So vis users like myself could easily contribute at least some money to the project, because I would happily do so. I just didn't know that the link existed, because it isn't mentioned on either the project home page or the github page.

**Comment by mojoaxel** [comment]
> Well it would be helpful if the opencollective link would be more prominent.

I've integrated it in the README (develop branch) but there has not been a single release since that. That's why the readme is not up to date...Somebody really should create a new release!

**Comment by avrahamcool** [comment]
`vis` is a great and impactfull project.
it has a large (and active) community - it will be a great loss if it will fade like that.
there are no real alternatives for the capabilities of `vis`.

will sponsoring the project really help? how much money are we talking here?
I think that we can raise the community awareness for that matter - it's a small price to pay for such a great project.

and also - it's time to thanks all the members of the project for their work so far.

**Comment by jgorene** [comment]
Indeed, it would be too bad if this project got lost because it really is a awesome library!
Unfortunately, I don't have enough skills to help effectively...

A single developer  (or almost in fact) to maintain this project must be huge over time!
All that remains is "to feed" the opencollective and find a sponsor...
Another solution would be to make a paid version (with low cost by user, hopeful;)... why not ?

**Comment by yotamberk** [comment]
If anybody is interested in Timeline, you are more than welcome using my branch:
https://github.com/yotamberk/timeline-plus/
I'm maintaining it and upgrading versions weekly.
I feel I had to branch off to my own fork because I couldn't handle the issues for the other modules. There was no easy way for me to maintain this lib without the separation... It seemed that the owners were not going to allow the separation, so I simply took it to my own fork - since I'm the only maintainer anyways...

**Comment by micahstubbs** [comment]
I might make my own fork for `Network` module if there isn't already another better one.  

agree with the other comments about the virtues of smaller, focused repos for the larger vis project.  has anyone on this thread seen someone else actively working on the `Network` module on their own Vis.Js fork?

**Comment by Jogai** [comment]
Dividing this into modules was discussed here: https://github.com/almende/vis/issues/2405

Did you talk to @yotamberk about centralizing in a new org? Because its nice to have a central point, but that only works if everyone is onboard...

**Comment by mojoaxel** [comment]
@micahstubbs Thanks for trying to help. Maybe an new organisation is not a bad idea! I see a few issues here:

First there is already an account [visjs](https://github.com/visjs) That would be perfect for that. Now that is still very early can be use [visjs](https://github.com/visjs) instead of [visjs-community](https://github.com/visjs-community)? I could give you all rights.

Also: We have this [OpenCollective account](https://opencollective.com/vis). At the moment it is connected to the almende repository. We should try to move it the the community-account.

Thoughts?

**Comment by micahstubbs** [comment]
@mojoaxel, ah, that's good to know!  I didn't find the visjs github account before.  If we agree on the broad goals of 

1) reviewing/unblocking all the open PRs 
2) smaller modules 
3) many collaborators to make 1) fast & easy

then I'm sure we could collaborate over there.

**Comment by micahstubbs** [comment]
@Jogai yea, had some emails with @yotamberk a few weeks ago.  he seemed focused on timeslines-plus and open-minded about the direction of the rest of the project.

**Comment by avrahamcool** [comment]
@micahstubbs, @mojoaxel
the visjs account sounds good - it's still early and we should migrate to this account instead of creating a new community.

@mojoaxel good work on https://github.com/almende/vis/issues/2405, I think this should be the guiding stone for the migration.

I would also add tasks for updating the typescript types.
I already forked defently-typed to visjs-community, and pushed some minor fixes to the vis types.

I'm very interested in the graph2D/timeline part of vis, and I'm willing to contribute time and effort for the community.
but I currently stuck - I don't even know where to start.
I'm not very familiar with the current code-base so i don't want to do damage.

something that is missing from the new fork is a repo for the dataset/dataview and other common utilities.

also, if we can have some sort of manual guide on how to build and test the project - so we can know that we  are not breaking anything while contributing.

**Comment by micahstubbs** [comment]
💯 

> also, if we can have some sort of manual guide on how to build and test the project - so we can know that we are not breaking anything while contributing.

**Comment by avrahamcool** [comment]
maybe it'll be beneficial to use tools like https://github.com/lerna/lerna for managing a mono repo of vis.

**Comment by Jogai** [comment]
See also https://github.com/korfuri/awesome-monorepo

**Comment by mojoaxel** [comment]
vis now is fully community managed:
* [x] The new home is now [visjs](https://github.com/visjs).
* [x] The library has been slit up in multible libraries and can now be maintained by different teams.
* [x] The website has moved: www.visjs.org

@micahstubbs please consider to contribute to [vis-network](https://github.com/visjs/vis-network)! We would love to welcome as a maintainer! I thing having both the [visjs](https://github.com/visjs) and [visjs-commnity](https://github.com/visjs-community) is very confusing.
